tests/test_securitygroup.py

```diff
@@ -162,3 +162,45 @@ def test_securitygroup_references(mock_properties, mock_details):
     assert 'myapp' in ingress
     assert '22' == ingress['myapp'][0]['start_port']
     assert '22' == ingress['myapp'][0]['end_port']
+
+
+@mock.patch('foremast.securitygroup.create_securitygroup.get_details')
+@mock.patch('foremast.securitygroup.create_securitygroup.get_properties')
+def test_merge_security_groups(mock_properties, mock_details):
+    """Make sure default Security Groups are added to the ingress rules."""
+    app_ingress = {
+        'test_app': [
+            {
+                'start_port': 30,
+                'end_port': 30,
+            },
+        ],
+    }
+
+    mock_properties.return_value = {
+        'security_group': {
+            'ingress': app_ingress,
+            'description': '',
+        },
+    }
+
+    default_sg = {
+        'myapp': [
+            {
+                'start_port': 22,
+                'end_port': 22,
+            },
+        ],
+        'test_app': [
+            {
+                'start_port': 31,
+                'end_port': 31,
+            }
+        ]
+    }
+
+    with mock.patch.dict('foremast.securitygroup.create_securitygroup.DEFAULT_SECURITYGROUP_RULES', default_sg):
+        sg = SpinnakerSecurityGroup()
+        ingress = sg.update_default_rules()
+        assert len(ingress['myapp']) == 1
```

Comment:
Check the values instead of just length. I have a feeling this might get messed up in the future and we'll want to make sure the values are good.

Reply:
Good point. Did that.